rsky-firehose/src/main.rs

```diff
@@ -146,9 +146,7 @@ async fn process(message: Vec<u8>, client: &reqwest::Client) {
                                                 author: commit.repo.to_owned(),
                                                 record: post
                                             };
-                                            if let Some(prev) = commit.prev {
-                                                create.prev = Some(prev.to_string());
-                                            }
+                                            create.prev = commit.prev.clone();
```

Comment:
could probably move this up into the initialization above; I took out the `if` since the variable we're setting is an `Option` too